lua/render-markdown/init.lua

```diff
@@ -128,8 +129,9 @@ function M.setup(opts)
             latex = '@markup.math',
             quote = '@markup.quote',
         },
+        enabled = false,
```

Comment:
The default value should be `true`.

The plugin should continue to function as it currently does by default and allow users to opt in to the new feature.